core/01_core.js

```diff
@@ -17,6 +17,7 @@
     Promise,
     ObjectFromEntries,
     MapPrototypeGet,
+    MapPrototypeHas,
```

Comment:
Why this change?

Reply:
It's a missing import used at https://github.com/denoland/deno/blob/d81c5b51b32247c96af78d90e89429e33b2a58ac/core/01_core.js#L102 Forgot that change was in my workspace for this, oh well.